torch/csrc/CudaIPCTypes.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <TH/TH.h>
```

Comment:
nit: I don't see any references to TH types in this header, so better to replace this with direct includes to what you use

Reply:
Done